dbms/src/Core/Block.cpp

```diff
@@ -514,6 +514,26 @@ static ReturnType checkBlockStructure(const Block & lhs, const Block & rhs, cons
     return ReturnType(true);
 }
 
+Block mergeBlocks(Blocks && result_blocks)
+{
+    auto & sample_block = result_blocks[0];
```

Comment:
Add comments if result_blocks can't be empty. or add non-empty check here.

Reply:
> Add comments if result_blocks can't be empty. or add non-empty check here.

done.